.github/workflows/doc.yml

```diff
@@ -3,7 +3,7 @@ name: gh-pages
 on:
   push:
     branches:
-      - master
+      - main
       - remove-travis
```

Comment:
This branch does not seem to exist any more? Maybe left over from transitioning to GH Actions?

Reply:
Yes, I think that can be removed. Good spot!

Reply:
Hoops, nice catch 👍🏼 , thanks